src/main/resources/static/index.html

```diff
@@ -16,8 +16,14 @@
     <link href="css/action.css" rel="stylesheet">
     <link href="css/ng-img-crop.css" rel="stylesheet">
 
+    <style type="text/css">
+        [ng\:cloak], [ng-cloak], .ng-cloak {
+            display: none !important;
+        }
+    </style>
+
 </head>
-<body>
+<body  class="ng-cloak" ng-cloak>
```

Comment:
Why using both attribute and class? It's enough to put only attribute on html level where you have ng-app directive